templates/master/00-master/baremetal/files/baremetal-get-vip-subnet-cidr.yaml

```diff
@@ -0,0 +1,30 @@
+filesystem: "root"
+mode: 0755
+path: "/usr/local/bin/get_vip_subnet_cidr"
```

Comment:
Ditto.

If you have a collection of Python helpers, maybe simpler to put them in one file with subcommands?

Reply:
That's a good suggestion. Thanks.